### PR TITLE
update nest logo

### LINKED ISSUE #71: Update all logos on teal.gallery home & demo page
@donyunardi 

Please can we get this done asap? We need to work on the NEST promo video and there will be screen recording using the teal.gallery site.

Feel free to grant access to any coredev member of the new logos.

### PR DESCRIPTION
**Pull request by donyunardi**
fixes #71 #66

one more thing to do is to update GIF.

### CODE REVIEW

**Comment by donyunardi** [comment]
I can't seem to make dynamic url for favicon: 
https://github.com/insightsengineering/teal.gallery/blob/bf2deaa45ac00e4e6477ae2745e0866de4d46af4/_internal/quarto/_quarto.yml#L35

Keeping it as is for now and updated the nest logo in the repo.

**Comment by donyunardi** [comment]
### Result

#### Home page
image

#### Demo page
Kapture 2023-09-13 at 17 03 01

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # teal.gallery
 
-<img align="right" width="100" height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/thumbs/teal.png">
+<img align="right" width=auto height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/teal.png" style="margin-left:10px;">
+<img align="right" width=auto height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/teal.gallery.png">
+
 
 A gallery of sample apps based on the [`teal` framework](https://github.com/insightsengineering/teal).
```
Comment (pawelru) [inline README.md:7]:
```suggestion
A gallery of sample apps based on the [`teal`](https://github.com/insightsengineering/teal) framework.
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # teal.gallery
 
-<img align="right" width="100" height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/thumbs/teal.png">
+<img align="right" width=auto height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/teal.png" style="margin-left:10px;">
+<img align="right" width=auto height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/teal.gallery.png">
```
Comment (pawelru) [inline README.md:4]: can we add `href` to redirect to appropriate packages? Now it just opens a new tab with logo

**Comment by pawelru** [comment]
These are good changes. Thank you!

When reviewing the changes I noticed one thing. Some of the apps were recorded with W: 896px | H: 445px and some in W: 3584px | H: 1832px. Previously we had everything in W: 970px | H: 555px so that was consistent. Any particular reason? I think we should have rather smaller as this will be relatively small.

File: README.md (hunk 2 above)
Comment (telepath37) [reply README.md:4]: agree with this comment - let's add a `href` to the `teal.gallery` package

Comment (donyunardi) [reply README.md:4]: good idea, will do.

**Comment by donyunardi** [comment]
The way it works now, the GIF and PNG are in `img` tags that's on top of each other. 
Theoretically, as long as the PNG covering the GIF, we should be good, but I understand if we want to make it an exact same size.

For the GIF, I wasn't able to follow the instruction to get the expected result. I wasn't sure if it caused by the different size of window or laptop screen, but after couple tries, I was aiming to have a somewhat low file size. All the GIFs consistently have size 896 × 456. Anything more than this, will cause me > 1 MB. 
If you want to record exactly at 1470x840, then you would have to size your browser exactly at that size before starting to record. It feels like a lot of work because I have to keep on trying to resize the browser to fit that size.

For the PNG, I made some update with the css so that the PNG will always fill the element that encapsulating the PNG. I like it a bit bigger because it has better resolution.

Let me know your thought, in the meantime, I'll recording the GIF again at the recommended size. I need to find a good tool to resize images too. I am on Mac so Previewer doesn't give option to resize.

**Comment by donyunardi** [comment]
okay, it turns out it's not that bad once I set my browser size correctly. 😄 
I should be able to quickly record the rest and fix the PNG size.
Stay tuned.

File: README.md (hunk 2 above)
Comment (donyunardi) [reply README.md]: done, added href to both logos.

**Comment by donyunardi** [comment]
@telepath37 @pawelru take 2 ready for review.

**Review by telepath37 — APPROVED** [review]

**Comment by averissimo** [comment]
@donyunardi the logos on the favicon look "squashed", due to the image not having a 1:1 canvas.

* left one is using a square 1:1 image size
* right one is the current [dev deployment](https://genentech.shinyapps.io/nest_safety_dev/)

image

We might need to create a new folder on [insightsengineering/hex-stickers](https://github.com/insightsengineering/hex-stickers) for favicons 

_(as well as replace the icons in thumbs directory with the new logos)_

**Comment by donyunardi** [comment]
You sir have a very sharp eye! 🦅 
Not a bad idea about favicons folder on hex-stickers. Maybe open an issue there?

I did mention about thumbs directory here:
https://github.com/insightsengineering/hex-stickers/issues/28

**Pull request merged**